### ISSUE TITLE
Non-admin users cannot remove ssh keys

### ISSUE THREAD

**Issue by jorihardman** [deis/deis#4167]
A continuation of closed issue #3285.

If a non-admin user uploads a new key, they cannot remove it with the client afterwards.

```
> deis keys:add
Found the following SSH public keys:
1) deis.pub jori@omnom.local
2) id_boot2docker.pub jori@omnom.local
3) id_rsa.pub jorihardman@gmail.com
4) master_hub_key.pub root@alarmpi
0) Enter path to pubfile (or use keys:add <key_path>)
Which would you like to use with Deis? 2
Uploading jori@omnom.local to Deis...done
jori@~/.deis
> deis keys
=== travis Keys
testing-worker-linux-docker-ad3a1a05-3371-linux-12 ssh-rsa AAAAB3Nz...73VryvZn5/
testing-worker-linux-docker-1efcdab6-3702-linux-6 ssh-rsa AAAAB3Nz...cZPtxH4MA1
testing-worker-linux-docker-c6476e6e-3388-linux-8 ssh-rsa AAAAB3Nz...Q7/a1nDy97
jori@omnom.local ssh-rsa AAAAB3Nz...wfoHTA3pft
testing-worker-linux-docker-d9467f71-3382-linux-3 ssh-rsa AAAAB3Nz...GmsHTzfQl/
testing-worker-linux-docker-24762f8c-3361-linux-15 ssh-rsa AAAAB3Nz...ievZq3yJCH
testing-worker-linux-docker-00fcdaa8-3698-linux-1 ssh-rsa AAAAB3Nz...341IUccveL
testing-worker-linux-docker-b552791d-3372-linux-8 ssh-rsa AAAAB3Nz...tF44S5ZVON
testing-worker-linux-docker-73fbc1db-3384-linux-16 ssh-rsa AAAAB3Nz...izeRuFuv3/
testing-worker-linux-docker-a352790b-3374-linux-16 ssh-rsa AAAAB3Nz...gI8wqFDAdR
testing-worker-linux-docker-aabe9502-3346-linux-11 ssh-rsa AAAAB3Nz...v/1VPv5Ylh
testing-worker-linux-docker-c5476e6d-3358-linux-7 ssh-rsa AAAAB3Nz...RJ+EZu962j
testing-worker-linux-docker-89dffe21-3357-linux-7 ssh-rsa AAAAB3Nz...3m5P9B6+fl
jori@~/.deis
> deis keys:remove jori@omnom.local
Removing jori@omnom.local SSH Key... 403 FORBIDDEN
Detail:
You do not have permission to perform this action.
```

**Comment by bacongobbler** [comment]
@jorihardman would you please test and confirm that #3931 resolves your issue? If not that's okay, v1.9 should be released next Tuesday which will include the patch.

**Comment by jorihardman** [comment]
Thanks for the quick followup guys! I should've looked at the latest fixes first. This certainly looks like it resolves my issue, but I will be sure to test 1.9 when it comes out!
